src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -32,13 +32,16 @@ public enum ErrorCode {
   EMBEDDING_PROVIDER_RATE_LIMITED(
       "The configured Embedding Provider for this collection is rate limiting your requests"),
 
-  EMBEDDING_PROVIDER_TIMEOUT("The configured Embedding Provider timed out."),
+  EMBEDDING_PROVIDER_TIMEOUT("The configured Embedding Provider timedout."),
```

Comment:
nit: should be "timed out." (i.e. don't change)

Reply:
Merge issue:(